modules/GKB/Release/Steps/DataExporter.pm

```diff
@@ -51,11 +51,6 @@ override 'run_commands' => sub {
             ["perl upload_europepmc.pl -version $version > upload_europepmc.out 2> upload_europepmc.err"]
         ]
     );
-
-    # Run hapmap
-    $self->cmd('Running hapmap script', [
-        ["./1haprefseq.pl -user $user -pass $pass -db $db > 1haprefseq.out 2> 1haprefseq.err"]
-    ]);
```

Comment:
Weren't we interested in keeping this still in case it's needed in the future? Would it make more sense to keep it here but commented out?

Reply:
It should be okay to remove it.  

If we decide to start producing the HapMap file again in the future, it would get re-written in Java to be part of the data-export module in the data-release-pipeline GitHub repo.

Reply:
OK so based on that, I assume the hapmap file produced is pretty simple to create?

Reply:
Yup, it's just a mapping of HapMap ids to Reactome top-level pathways